src/utils/plotSpecs/generateEmbeddingCategoricalSpec.js

```diff
@@ -4,7 +4,7 @@ import { getAllCells, getSampleCells } from 'utils/cellSets';
 
 const paddingSize = 5;
 
-const generateSpec = (config, plotData, cellSetLegendsData) => {
+const generateSpec = (config, method, plotData, cellSetLegendsData) => {
```

Comment:
Why can't we use the config and need to pass a separate parameter now?

Reply:
Using this to generate the default axis names, instead of dispatching a redux action `updateEmbeddingAxesIfNecessary` every time, like it was done before.